include/erlcloud_ddb_streams.hrl

```diff
@@ -36,8 +36,24 @@
 -record(ddb_streams_describe_stream,
         {stream_description :: #ddb_streams_stream_description{}
         }).
+-record(ddb_streams_record_dynamodb,
```

Comment:
This record name should be `ddb_streams_stream_record` according to the [document](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_StreamRecord.html), I've changed it.